src/create-app.js

```diff
@@ -37,6 +39,8 @@ module.exports = (name, options) => {
      * @param {function} handler - Handler to be called when app is started without intent
      */
     app.onStart = (handler) => {
+        info('Setted up onStart handler');
+        debug(`Setted up onStart handler: "${handler}"`);
```

Comment:
these `Setted / Created up xyz ...` is unnecessary. It is sufficient to show info/debug messages only when request is received and when response is generated